typescript/secrets-manager-secret-rotation/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "secrets-manager-secret-rotation",
+  "version": "1.0.0",
+  "description": "Shows how to use a custom resource",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "cdk": "cdk"
+  },
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com",
+    "organization": true
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@aws-cdk/assert": "*",
+    "@types/jest": "^26.0.10",
+    "@types/node": "10.17.27",
+    "aws-cdk": "*",
+    "jest": "^26.4.2",
+    "ts-jest": "^26.2.0",
+    "ts-node": "^9.0.0",
+    "typescript": "~3.9.7"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-ec2": "*",
+    "@aws-cdk/aws-elasticache": "*",
+    "@aws-cdk/aws-iam": "*",
+    "@aws-cdk/aws-lambda": "*",
+    "@aws-cdk/aws-secretsmanager": "*",
+    "@aws-cdk/aws-sns": "*",
+    "@aws-cdk/aws-sns-subscriptions": "*",
+    "@aws-cdk/aws-sqs": "*",
+    "@aws-cdk/core": "*"
+  }
```

```suggestion
  "dependencies": {
    "aws-cdk-lib": "*",
    "constructs": "*",
  }
```

Would you mind upgrading this to CDK v2 for consistency with the other projects we have? I've added code suggestions with all the necessary code to upgrade. It's possible I may have missed something though but let me know if you have trouble.

By the way you'll need to update the lockfile once these suggestions are accepted. Run `npm install` and commit the updated lockfile changes.

---

Certainly, I'll upgrade this to v2 and commit the changes.

---

Done and done!